builtin/logical/pki/cert_util.go

```diff
@@ -149,6 +149,7 @@ func fetchCAInfo(ctx context.Context, b *backend, req *logical.Request, issuerRe
 func fetchCertBundle(ctx context.Context, b *backend, s logical.Storage, issuerRef string) (*issuerEntry, *certutil.CertBundle, error) {
 	if b.useLegacyBundleCaStorage() {
 		// We have not completed the migration so attempt to load the bundle from the legacy location
+		b.Logger().Info("Using legacy CA bundle")
```

Comment:
Is this enough information for an operator to know why this log line is emitted?

Reply:
Changed to 

`b.Logger().Info("Using legacy CA bundle as PKI migration has not completed.")`